src/generated/resources/data/wotr/recipe/key_forge.json

```diff
@@ -2,16 +2,18 @@
   "type": "minecraft:crafting_shaped",
   "category": "misc",
   "key": {
-    "E": "minecraft:ender_pearl",
-    "a": "#minecraft:anvil"
+    "L": "#minecraft:overworld_natural_logs",
```

Comment:
Looks like you didn't update the data gen, so this will just be overwritten next time that is run. See `com.wanderersoftherift.wotr.datagen.ModRecipeProvider`